### DISCUSSION TITLE
Disable formatter fort single file

### DISCUSSION THREAD

**Discussion by decapo01** [gleam-lang/gleam#2709] [Questions & support]
I'm messing around with the lustre library that makes html.  I would like to format that in a different way so it looks more like html but the gleam auto formatter is changing the code around.  How would I disable the auto formatter for a single file?

**Comment by lpil** [comment]
The configure doesn't support any configuration, you'll have to not use the formatter if you don't want to use the Gleam formatting.

**Reply by decapo01** [reply]
Ah that's cool, maybe one day.  This language is awesome, great work!
